scripts/ingest_docs.py

```
@@ -67,15 +111,23 @@ def main():
+        elif suffix == ".docx":
+            text = extract_text_from_docx(file_path)
+        elif suffix == ".html":
+            text = extract_text_from_html(file_path)
+        elif suffix in [".txt", ".md", ".text"]:
             text = file_path.read_text(encoding="utf-8")
-        except Exception as e:
-            logger.error("Failed to read text file: %s", e)
+        else:
+            logger.error("Unsupported file format: %s. Supported formats: PDF, DOCX, HTML, TXT, MD", suffix)
             sys.exit(1)
+    except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)